CHANGELOG.md

```diff
@@ -142,6 +142,7 @@ CHANGELOG
   - Changes feed URL to JSON format (contains more information).
   - Adds fields from the JSON feed.
 - `intelmq.bots.parsers.generic.parser_csv`: Parameter `type` is deprecated, `default_fields` should be used. (PR#2293 by Filip Pokorný)
+- `intelmq.bots.parsers.generic.parser_csv`: Parameter `skip_header` now allows also integer as a fixed number of lines to skip. (PR#2313 by Filip Pokorný)
```

Comment:
I would write: "allows allows an integer as value, denoting the number of initial lines to skip" (grammar, but .. that's maybe nitpicking now :) )